merlin/study/step.py

```diff
@@ -43,6 +43,7 @@
 
 
 LOG = logging.getLogger(__name__)
+QUEUE_TAG = "[merlin]_"
```

Comment:
Luc explained the purpose of this to me - I think it might be a good idea if this is dynamically linked to a version number, so like [merlin_1.8], for two reasons: if we need to come back and modify this again in the future, it would be ideal if we don't collide with the same problem again and have a built-in mechanism to not be tied to ; might just be good from a backwards compatibility perspective to decouple. Version number may not be the best thing to hash/distinguish on, but worth some thought.

Reply:
Interesting idea. One problem with adding the version would be that users' old queues wouldn't work after an update. This update already will basically throw out old task queues, but that's necessary for using the new task priority (old queues break because they weren't set up to support priority). Tangential idea, maybe we can add a flag `omit-queue-tag` or something if a user wants to run without this.

Reply:
alternatively, we could have this be a config setting (in the config file) with  default value that we change when we need to. then if we get a corner case where somebody really needs to go back and do something else, they can, without us having to explicitly check for it

Reply:
i kind of like this approach: have QUEUE_TAG be a configuration setting. most users will be fine with the default, which we can change as we see the need to do so, but also gives super-users the chance to revert or do something more wild for their own needs

Reply:
Sounds good, I will implement this.

Reply:
@lucpeterson I just added support for queue_tag in the config. The vpn from my personal machine is finicky so I haven't tested the distributed version of this, but let me know if this follows your feature idea.